sherlock/notify.py

```diff
@@ -155,6 +155,33 @@ def start(self, message):
               Fore.GREEN + f"] {title}" +
               Fore.WHITE + f" {message}" +
               Fore.GREEN + " on:")
+        # An empty line between first line and the result(more clear output)
+        print('\r')
+
+        return
+
+    def finish(self, message="Results have been uploaded"):
```

Comment:
We don't upload anything. so it's better if have a better sentence here. 
like: `The processing has been finished.`

Reply:
I change it and also changes finish to end. Looks better to me if you dont agree i make it 'finish' again